sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -55,22 +60,21 @@ def plot_partial_dependence(
 
     .. note::
 
-        :func:`plot_partial_dependence` does not support using the same axes
-        with multiple calls. To plot the the partial dependence for multiple
-        estimators, please pass the axes created by the first call to the
-        second call::
+        :func:`PartialDependenceDisplay.from_estimator` does not support using
```

Comment:
I think that we want to revert this part. We still want the documentation to be unchanged.

Reply:
We only need a `.. deprecated:: 1.0` above the note mentioning to use the `.from_estimator` instead.